### ISSUE TITLE
[Feat] Provide the ability to configure sheet colours

### ISSUE THREAD

**Issue by Pjb518** [Pjb518/FoundryVTT-Level-Up-Official#138]
We should provide client settings for modifying the colours of the actor and item sheets. Some presets should be provided, and there should be a way for modules to register their own presets. This will be useful for modules that want to provide a thematic sheet option, for example.

This is currently be help up by a PR for Svelte here: https://github.com/sveltejs/svelte/pull/8275

This is in turn preventing a colour picker component making it into svelte-standard, which is a component library for TRL (the package we use to develop the system).

#30 also needs to be completed before this issue can be started.

**Comment by typhonrt** [comment]
You can actually enable the color picker component now, but have to use an override for loading a forked version of Svelte and uncomment a line in `svelte-standard` to enable the export of the picker component.

Here is a pict of the location in `node_modules` under `@typhonjs-fvtt/svelte-standard` that you have to uncomment:
<img width="881" alt="uncomment picker" src="https://user-images.githubusercontent.com/311473/226937995-8e6399a0-ada8-405c-8c99-bb6d16457126.png">

Here is the override in package.json:
<img width="491" alt="Svelte override" src="https://user-images.githubusercontent.com/311473/226938159-2ef7049f-30fe-4b9e-9899-5032d5abb10b.png">

To get this to install correctly with an existing repo you probably should delete the `node_modules` folder and `package-lock.json`. Modify the `package.json` adding the override / removing `svelte` from `dependencies`. Run `npm install` then uncomment the line in `svelte-standard` to enable the color picker component.

I'm presently working on a theme-editor w/ CSS variable support and automatic serialization & import / export that may come in handy. I should have that done in early April.

**Comment by Pjb518** [comment]
Thanks for info. If the Svelte PR stays on hold by time we get around to working on this, we'll definitely make use of the forked Svelte version.

Rather embarrassing that this ended up getting linked in the PR. I copied a full URL for the PR for information purposes, and GitHub, being too smart for its own good, realised that it could tie the two pages together 😄

**Comment by typhonrt** [comment]
I think it's fine in the PR.. Shows that people are interested in the feature; I wouldn't have seen it otherwise as I check up on the PR from time to time. Yeah You have to use `` when pasting URLs make it a comment.

**Comment by Pjb518** [comment]
Yeah, I just didn't even think about the fact it would link across repositories like that. Should have been obvious, really.

Anyway, I hope the PR gets merged in soon. We've been watching it closely for a little while now. 🤞
